redisson-hibernate/redisson-hibernate-4/src/main/java/org/redisson/hibernate/RedissonRegionFactory.java

```diff
@@ -189,6 +190,14 @@ public long nextTimestamp() {
         }
     }
 
+    private String qualifyName(String name) {
```

Comment:
I think it should be applied to 53 and 6 modules only since others don't handle qualifyName in RedissonRegionFactory at all.